running-locally.adoc

```diff
@@ -1,64 +1,46 @@
 [[running-the-demo-raincatcher-solution-locally]]
-Running The Demo Raincatcher Solution Locally
----------------------------------------------
-
-[[overview]]
-Overview
-~~~~~~~~
-
-This guide presents the steps necessary to:
-
-* Run the Raincatcher demo solution locally.
-* Edit Raincatcher modules and have the results reflected in the demo
-solution.
+= Running The Demo Raincatcher Solution Locally
 
 [[requirements]]
-Requirements
-~~~~~~~~~~~~
+== Requirements
 
 [[rhmap]]
-RHMAP
-~~~~~
+=== RHMAP
 
-The demo solution currently requires an instance of RHMAP and will not work properly without it. It can be a local or remote instance of RHMAP, but must be configured as outlined in the Raincatcher link:getting-started.adoc[Getting Started Guide]. This allows the local *raincatcher-demo-cloud* app to access
```

Comment:
You're right - this is probably fine. Given that we say it can be a local instance of RHMAP, I think the title isn't misleading.